alpha/alpha-fsm/src/main/java/org/apache/servicecomb/pack/alpha/fsm/channel/AbstractActorEventChannel.java

```diff
@@ -42,6 +45,7 @@ public void send(BaseEvent event) {
       this.sendTo(event);
       metricsService.metrics().doEventAccepted();
     } catch (Exception ex) {
+      logger.error("send Exception = [{}]", ex);
```

Comment:
log error can show the stack trace if the last object is exception,  but I'm not sure if it works for your case.

Reply:
Which reminds me, here should throw an exception